components/playground/instance/tiflash.go

```diff
@@ -84,17 +84,6 @@ func (inst *TiFlashInstance) Start(ctx context.Context, version v0manifest.Versi
 		return err
 	}
 
-	if binPath == "" {
-		installedVersion, err := profile.SelectInstalledVersion("tiflash", version)
-		if err != nil {
-			return err
-		}
-		dir, err := profile.BinaryPath("tiflash", installedVersion)
```

Comment:
this will need to read the v0 matifest.
and it will use "tiup install **" to install.
so... if user use v1 tiup and old version playground, I think this will cause error, but nothing we can do, need to upgrade the playground version